ctapipe/image/muon/muon_integrator.py

```diff
@@ -68,7 +68,8 @@ def __init__(self, mirror_radius, hole_radius, pixel_width=0.2,
         self.prediction = 0
         self.minlambda = 300.e-9
```

Comment:
if these are wavelengths, please require them to be astropy `Quantities`, so the unit is attached (e.g. define them as `300e-9 * u.m` or even `30 * u.angstrom`.